invenio_sipstore/api.py

```diff
@@ -244,7 +244,9 @@ def create(cls, pid, record, archivable, create_sip_files=True,
         """
         files = record.files if create_sip_files else None
         mtype = SIPMetadataType.get_from_schema(record['$schema'])
-        metadata = {mtype.name: json.dumps(record.dumps())}
+        metadata = {mtype.name: json.dumps(record.dumps(),
+                                           indent=4,
```

Comment:
True... and as it is stored in the database, this is bad :/
Should we keep creating files without indent or should we add a smart way to write them, as the SIPs may be stored in a compressed format? (we can have an option)

This was just an idea but if it is not relevant, we can skip it ^^